lang/de/miquiz.php

```diff
@@ -13,10 +13,12 @@
   $string['modulename_help'] .= $additional_info;
 }
 
+$string['miquiz_index_title_overview'] = $instance_name .' Overview';
 $string['miquiz_index_title'] = $instance_name .' Kurs &Uuml;bersicht';
 $string['miquiz_index_reports'] = 'Anzahl Reports';
 $string['miquiz_index_download'] = 'Statistiken herunterladen';
 $string['miquiz_index_noquizselected'] = 'Bitte w&auml;hlen sie mindestens ein Quiz aus!';
+$string['miquiz_index_overview'] = 'Zur Übersicht';
```

Comment:
We already have an overview here, so a better button name would be "Zur Gesamtübersicht" or "Zur globalen Übersicht"